### DISCUSSION TITLE
Per world for entity can use portals

### DISCUSSION THREAD

**Discussion by IceWaffles** [PurpurMC/Purpur#1210] [__Completed__] [answered]
Allow use of per world configuration for the option `entities-can-use-portals` in purpur.yml. For example, if that option is only disabled for the end, then entities (apart from players, obviously) can only not use end portal to go back into the overworld, but entities can still pass through nether portals from the overworld to the nether, as that option is not disabled for the overworld.

**Comment by BillyGalbreath** [comment, marked as answer]
It's already a per-world setting...
